build.gradle.kts

```
@@ -77,17 +82,34 @@ tasks.withType(JavaCompile::class) {
 }
 
 tasks.withType(Jar::class) {
-    archiveBaseName.set("examplemod")
+    archiveBaseName.set(modid)
     manifest.attributes.run {
         this["FMLCorePluginContainsFMLMod"] = "true"
         this["ForceLoadAsMod"] = "true"
 
         // If you don't want mixins, remove these lines
         this["TweakClass"] = "org.spongepowered.asm.launch.MixinTweaker"
-        this["MixinConfigs"] = "mixins.examplemod.json"
+        this["MixinConfigs"] = "mixins.$modid.json"
     }
 }
 
+tasks.processResources {
+    inputs.property("version", project.version)
+    inputs.property("mcversion", mcVersion)
+    inputs.property("modid", modid)
+    inputs.property("mixinGroup", mixinGroup)
+
+    //replace stuff in mcmod.info, nothing else
+    from(sourceSets["main"].resources.srcDir("resources")) {
+        include("mcmod.info")
+        include("mixins.$modid.json")
+
```

This is including a duplicate copy of both `mcmod.info` and `mixins.$modid.json`.
```suggestion
    filesMatching(listOf("mcmod.info", "mixins.$modid.json")) {
```

Additionally I think you are missing the actual placeholders in `src/main/resources/mcmod.info` that would be replaced by this operation:

```json
[
  {
	"modid": "${modid}",
	"name": "Xample Mod",
	"description": "A mod that is used as an example.",
	"version": "${version}",
	"mcversion": "1.8.9",
	"url": "https://github.com/romangraef/Forge1.8.9Template/",
	"updateUrl": "",
	"authorList": [
	  "You"
	],
	"credits": "",
	"logoFile": "",
	"screenshots": [],
	"dependencies": []
  }
]
```